extensions/iconify/package.json

```diff
@@ -22,10 +22,17 @@
       "subtitle": "Iconify",
       "description": "View available icons",
       "mode": "view"
+    },
+    {
+      "name": "search-icons",
```

Comment:
would it make sense to just add an `All` item in the `searchBarAccessory` of the other command? 🤔

Reply:
Hmm. Possible, but dunno. They behave differently both from the implementation and UX perspective.